.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -1,224 +1,283 @@
-
-      # Build aab release bundle
-      - android-build@0:
-          inputs:
-            - variant: release
-            - build_type: aab
-            - module: "$MODULE"
-
-      # Upload to Google Play Store
-      - google-play-deploy@3:
-          inputs:
-            - package_name: "$PACKAGE_NAME"
-            - app_path: "$BITRISE_AAB_PATH"
-            - service_account_json_key_path: "$BITRISEIO_SERVICE_ACCOUNT_JSON_KEY_URL"
-            - track: alpha
-
-      - cache-push@2: {}
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    
+    - git-clone@6: {}
+    
+    - cache-pull@2: {}
+    
+    - install-missing-android-tools@2:
+        inputs:
+        - gradlew_path: "$PROJECT_LOCATION/gradlew"
+    
+    # Execute Lint check
+    - android-lint@0:
+        inputs:
+        - project_location: "$PROJECT_LOCATION"
+        - module: "$MODULE"
+        - variant: "$VARIANT"
+    
+    # Execute UnitTest
+    - android-unit-test@1:
+        inputs:
+        - project_location: "$PROJECT_LOCATION"
+        - module: "$MODULE"
+        - variant: "$VARIANT"
+    
+    # Build aab release bundle
+    - android-build@0:
+        inputs:
+        - variant: release
+        - build_type: aab
+        - module: "$MODULE"
+    
+    # Upload to Google Play Store
+    - google-play-deploy@3:
+        inputs:
+        - package_name: "$PACKAGE_NAME"
+        - app_path: "$BITRISE_AAB_PATH"
+        - service_account_json_key_path: "$BITRISEIO_SERVICE_ACCOUNT_JSON_KEY_URL"
+        - track: alpha
+    
+    - cache-push@2: {}
+  
+  # Workflow not used for now
+  primary:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    
+    - git-clone@6: {}
+    
+    - cache-pull@2: {}
+    
+    - install-missing-android-tools@2:
+        inputs:
+        - gradlew_path: "$PROJECT_LOCATION/gradlew"
+    
+    # Execute Lint check
+    - android-lint@0:
+        inputs:
+        - project_location: "$PROJECT_LOCATION"
+    
+    # Execute Detekt
+    - gradle-runner@1:
+        inputs:
+        - gradlew_path: "$PROJECT_LOCATION/gradlew"
+        - gradle_task: detekt
+        title: Detekt
+    
+    # Execute UnitTest
+    - android-unit-test@1:
+        inputs:
+        - arguments: jacocoTestReport
+        - variant: "$VARIANT"
+        - module: "$MODULE"
+        - report_path_pattern: "*build/reports"
+        - project_location: "$PROJECT_LOCATION"
+    
+    # Upload all available result such as Lint, Detekt...
+    - deploy-to-bitrise-io: {}
+    
+    # Upload jacocoTestReport to Apps & Artifacts for later downloading
+    - deploy-to-bitrise-io@1:
+        inputs:
+        - deploy_path: "$PROJECT_LOCATION/$MODULE/build/reports/jacoco/jacocoTestReport/html/index.html"
+        - is_enable_public_page: 'false'
+        - is_compress: 'true'
+        - notify_user_groups: none
+  
+  # Workflow on release/* branch
+  release:
```

Comment:
any reason why do we need to re-arrange the order of workflows here? I think the previous order is good to keep: primary, develop, release, master.

Reply:
oh it is generated by Bitrise when applying and modifying on the web editor view, please just ignore 🙏

Reply:
Same to what I explained above, we should manage it here, try to not follow all the results from the Editor please 🙏 
Reformatting or reordering should be done in a separate task or PR I think.